app/main/views/direct_award.py

```diff
@@ -15,6 +15,20 @@
     get_valid_page_or_1, validate_and_return_updater_request, drop_all_other_fields)
 
 
+def get_project_by_id_or_404(project_id: int):
+    """During the transitional period, project_id may be either the internal or the public ID of the project."""
+    max_project_id = db.session.query(func.max(DirectAwardProject.id)).first()[0]
```

Comment:
Do we even need this check?  Could just try to fetch by ID, if no project found then try for external ID or 404?

Reply:
Guess not. I'll add a try/catch around the query instead.